pets/cassandra/on-start.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+# Parse out hostname, formatted as: petset_name-index
+IFS='-' read -ra ADDR <<< "$(hostname)"
+MY_ID=$(expr "1" + "${ADDR[1]}")
+CFG=/etc/cassandra/cassandra.yaml
+
+i=0
+while read -ra LINE; do
+    let i=i+1
+    if [ $i = $MY_ID ]; then
+        MY_NAME=$LINE
+    fi
+    PEERS=("${PEERS[@]}" $LINE)
+done
```

Comment:
I want a generic thing that works across all pets (so not java code, and definitey without a kube dependency if we can avoid it). The peer finder that invokes this script does so based on resolving DNS records created on behalf of the petset. I'm fine using an equivalent java program, if you'd like to check one into 2 places (kube and some C* project that only accepts java code).